Documentation/ExtensionArchitecture/FileStructure/_ext_localconf.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+// all use statements must come before the definition check
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
+// Prevent Script from being called directly
+defined('TYPO3') or die();
+
+// Add your code here
+ExtensionManagementUtility::doSomething();
```

Comment:
Also here: ExtensionManagementUtility does not have a doSomething method?

Reply:
same answer as above

Reply:
same suggestion as above ;-)

Reply:
Sorry to bother, see no change or reaction?